backport/backport.py

```diff
@@ -14,53 +14,59 @@
 
 from etl.db import get_engine
 from etl.files import checksum_str
-from etl.grapher_model import (GrapherConfig, GrapherDatasetModel,
-                               GrapherSourceModel, GrapherVariableModel)
+from etl.grapher_model import (
+    GrapherConfig,
+    GrapherDatasetModel,
+    GrapherSourceModel,
+    GrapherVariableModel,
+)
 
 WALDEN_NAMESPACE = os.environ.get("WALDEN_NAMESPACE", "backport")
 
 log = structlog.get_logger()
 
+engine = get_engine()
 
-def _walden_values_metadata(
-    ds: GrapherDatasetModel, short_name: str, origin_md5: str
-) -> WaldenDataset:
+# preload walden catalog to improve performance (initializing the catalog takes a while)
+walden_catalog = WaldenCatalog()
+
+
+def _walden_values_metadata(ds: GrapherDatasetModel, short_name: str) -> WaldenDataset:
     """Create walden dataset for grapher dataset values.
     These datasets are not meant for direct consumption from the catalog, but rather
     for postprocessing in etl.
     :param short_name: short name of the dataset in catalog
-    :param origin_md5: MD5 hash of data values in grapher used to decide whether to recompute
-        or not in the next run
     """
     return WaldenDataset(
         namespace=WALDEN_NAMESPACE,
         short_name=f"{short_name}_values",
         name=ds.name,
+        date_accessed=dt.datetime.utcnow().isoformat(),
         description=ds.description,
         source_name="Our World in Data",
```

Comment:
I think we should changes this to "Our World in Data catalog backport" or something like that to make it clear that we are not the original source and one should inspect the sources of the config json file to understand the upstream sources

Reply:
Good idea, fixed.